WcaOnRails/app/views/registrations/psych_sheet_event.html.erb

```diff
@@ -5,7 +5,10 @@
     <%= cubing_icon @event.id %> <%= @event.name %> Psych Sheet
   </h2>
   <p>
-  Sorted by: <%= @psych_sheet.sort_by %>, sort by <a href="<%= competition_psych_sheet_event_path(@competition, @event.id) %>?sort_by=<%= @psych_sheet.sort_by_second %>"><%= @psych_sheet.sort_by_second %></a>
+  Sorted by: <%= @psych_sheet.sort_by %>
+  <% if @event.id != "333mbf" %>
+  , sort by <a href="<%= competition_psych_sheet_event_path(@competition, @event.id) %>?sort_by=<%= @psych_sheet.sort_by_second %>"><%= @psych_sheet.sort_by_second %></a>
```

Comment:
I understand that you're copying something that has been there before, but touching this part of the code could be a nice incentive to clean up this ugly hack.

Rails has a `link_to` helper which generates the `a href` tags for you and which can also support query parameters: https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to